apps/user_group/models.py

```diff
@@ -33,9 +33,9 @@ def get_for(user):
 
     @staticmethod
     def get_for_member(user):
-        return UserGroup.objects.filter(
-            members=user
-        ).distinct()
+        return UserGroup.objects.annotate(
+            members_count=models.Count('members', distinct=True)
```

Comment:
Was members not used before?

Reply:
Not sure what you meant. But the filter is done by `.filter(members=user)`. @k9845 is only adding annotate here (members_count).

Reply:
I too am not sure what I mean, but just making sure this doesn't break the previous API usage.

Reply:
@k9845 Could you add a test if not present for the filter behavior.

Reply:
This is done @AdityaKhatri